spec/dzt/tiler_spec.rb

```diff
@@ -20,10 +20,12 @@
         image = Magick::Image::read("#{tmpdir}/11/1_1.jpg").first
         expect(image.columns).to eq(512)
         expect(image.rows).to eq(512)
+        expect(image.quality).to eq(75)
         # edge
         image = Magick::Image::read("#{tmpdir}/11/2_2.jpg").first
         expect(image.columns).to eq(168)
         expect(image.rows).to eq(443)
+        expect(image.quality).to eq(75)
```

Comment:
This was previously the ImageMagick default of `92` due to the quality not getting correctly set when saving.